packages/governance/src/contractGovernance/governParam.js

```diff
@@ -34,30 +35,6 @@ const makeParamChangePositions = changes => {
   return harden({ positive, negative });
 };
 
-/** @type {ValidateParamChangeQuestion} */
-const validateParamChangeQuestion = details => {
```

Comment:
Great to see this disappear! But the type still needs to be deleted from types.js